### PR TITLE
changed port page

### PR DESCRIPTION
**Pull request by EllaFurman**
Updated port.md

### CODE REVIEW

**Comment by EllaFurman** [comment]
Done :)

Ella Furman

Marketing Manager

Website - getport.io

Checkout our DevEx Community
<https://join.slack.com/t/devex-community/shared_invite/zt-1ksuwt1ir-7QkIIe5Ue3PXdun9YUWo3w>

Sent via Superhuman ***@***.***>

**Review by FL303 — APPROVED** [review]

File: content/ecosystem/platform-tooling/developer-portals/port.md [hunk 1]
```diff
@@ -1,106 +1,58 @@
-+++
-title="Port"
-aliases="/frameworks/port"
-url="/developer-portals/port"
-+++
```
Comment (christophcrichter) [inline content/ecosystem/platform-tooling/developer-portals/port.md:5]: Please add this part back in. This is needed for general navigation. Thanks.

**Review by christophcrichter — CHANGES_REQUESTED** [review]
Please see my comment regarding the header.

**Comment by EllaFurman** [comment]
Done :)

Ella Furman

Marketing Manager

Website - getport.io

Checkout our DevEx Community
<https://join.slack.com/t/devex-community/shared_invite/zt-1ksuwt1ir-7QkIIe5Ue3PXdun9YUWo3w>

Sent via Superhuman ***@***.***>

**Review by christophcrichter — APPROVED** [review]
LGTM; thanks for contributing

**Comment by EllaFurman** [comment]
Thank you!
I put in another addition now :)

Ella Furman

Marketing Manager

getport.io

Join our Port Community
<https://join.slack.com/t/port-community/shared_invite/zt-1wdjwizkn-0i7g9LQeQ4rdU5_YSe6vDA>


Sent via Superhuman ***@***.***>

**Pull request merged**